packages/raml-generator/src/cli.js

```diff
@@ -0,0 +1,55 @@
+/** Based on this blog - https://www.twilio.com/blog/how-to-build-a-cli-with-node-js */
```

Comment:
I would like that we use the same setup for our CLI scripts. For example, see the broken-link-checker.

We usually use `mri` for parsing command arguments. For small CLI this is perfectly sufficient, as the library is also very small compared to e.g. `arg` and `inquirer`.
Also, `inquirer` is not really necessary for this use case.

Reply:
I don't understand what you mean by inquirer is not necessary for this use case.

Reply:
It's not necessary to prompt the user to input the value, when you can simply pass them as command arguments.

Prompting usually is used for things like generating or initializing a project or something, where the user needs to provide multiple information.

Reply:
> I would like that we use the same setup for our CLI scripts. For example, see the broken-link-checker.

I don't understand what you mean, it doesn't appear to me that the setup should be similar, at least one main reason is broken link checker does not take arguments.

Reply:
I'm referring to the part where you parse the arguments, as this is the only case where a library is useful that does that for you.

https://github.com/commercetools/commercetools-docs-kit/blob/14581f1d3d715de2cdf0d3e231b7f49777c8aa13/packages/broken-link-checker/bin/broken-link-checker.js#L11-L36

You can also check in the appkit, as we use it in all our scripts: https://github.com/commercetools/merchant-center-application-kit/search?q=mri&unscoped_q=mri

Reply:
I use `mri` now.

Reply:
Thanks but we should also remove `inquirer`, as I explained before

Reply:
From my perspective, `mri` does not perform the same function as `inquirer`, so I disagree when why you say `inquirer` is not necessary.

Reply:
So how do you specify optional arguments?

Reply:
Check here for an example: https://github.com/commercetools/merchant-center-application-kit/blob/d4273e55d3de7e99d7f4bba6aeaf786a2bbf63a8/packages/create-mc-app/bin/create-mc-app.js#L14-L33

Probably it makes sense to specify the src/dest paths as options:

```
commercetools-ramldoc-generator [spec_name] [options]

--src   <path> The path to the source file
--dest <path> (optional) The path to the destination file
```

The `dest` option has a default value, if none is provided.